src/lib/Ska/Starcheck/Obsid.pm

```diff
@@ -1074,9 +1075,8 @@ sub check_star_catalog {
     my $min_z = $z_ang_max;
 
     my $dither;			# Global dither for observation
-    if (defined $self->{DITHER_Y_AMP} and defined $self->{DITHER_Z_AMP}) {
-	$dither = ($self->{DITHER_Y_AMP} > $self->{DITHER_Z_AMP} ?
-		   $self->{DITHER_Y_AMP} : $self->{DITHER_Z_AMP}) * 3600.0;
+    if (defined $self->{cmd_dither_y_amp} and defined $self->{cmd_dither_z_amp}) {
+	$dither = max($self->{cmd_dither_y_amy}, $self->{cmd_dither_z_amp});
```

Comment:
I'm not sure if there are two acquisitions in the MON case or large dither setup.  I'll look at that again.  In general, by two catalogs, I just meant independently checking the acquisition stars and the guide/fid sets as two different 8-object sets.